examples/gke-security-best-practices/builder/platform-uploads/setup-scripts/01-acme-customizations.sh

```diff
@@ -0,0 +1,66 @@
+# Copyright 2018 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+#!/bin/bash -xe
+
+# skip interactive post-install configuration steps
+export DEBIAN_FRONTEND=noninteractive
+
+sudo apt-get update -y
+sudo apt-get install -y curl software-properties-common
+sudo apt-add-repository -y ppa:ansible/ansible
+
+# Install kubectl
```

Comment:
Why not install via gcloud? Actually why not just use the version baked into gcloud?

Reply:
wasn't aware of it :-) I will address. You are right, that is a best practice, using the gcloud one ensures that it will be upgraded over time.